binding/binding_nomsgpack.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2014 Manu Martinez-Almeida.  All rights reserved.
```

Comment:
1. please update copyright
2. filename: `binding_nomsgpack.go` `binding_msgpack_test.go`, should we use `xxx_msgpack_xxx`?
3. the file and `binding.go` have repeat code, should we use reduce?
thanks!

Reply:
```suggestion
// Copyright 2020 Gin Core Team. All rights reserved.
```

I will have fresh look on removing/refactor duplicate code later.